.changes/next-release/service.s3-bugfix-1615945295096775000.json

```diff
@@ -0,0 +1,9 @@
+{
+ "ID": "service.s3-bugfix-1615945295096775000",
+ "SchemaVersion": 1,
+ "Module": "service/s3",
+ "Type": "bugfix",
```

Comment:
Is this bugFix type intentional?

Reply:
Yeah, the changetool doesn't have `documentation` type, `announcement` doesn't really fit well, but could use that instead of `bugfix`.

Reply:
Yeah annoucement isn't a fit either since it does not have a version bump associated with it.

given this `bugfix` is still the best type unless we also add `documentation` type.